workflows/isoseq.nf

```diff
@@ -133,10 +137,18 @@ workflow ISOSEQ {
         .groupTuple()
         .set { ch_tcollapse }
 
-    GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
+    if (params.capped == true) {
+        GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
         ch_tcollapse,
         Channel.value("capped"),
         Channel.value("1,1,1"))
+    }
+    else {
+        GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
+        ch_tcollapse,
+        Channel.value("no_cap"),
+        Channel.value("1,1,1"))
+    }
```

Comment:
```suggestion
    cap_value = params.capped == true ? Channel.value("capped") : Channel.value("no_cap")

    GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
    ch_tcollapse,
    cap_value,
    Channel.value("1,1,1"))
```

Reply:
I let you choose if you want to rewrite that part or not.

Reply:
Thanks for the suggestion!